pupil_src/shared_modules/pupil_producers.py

```diff
@@ -302,21 +301,24 @@ def start_eye_process(self, eye_id):
             for ext in (".mjpeg", ".mp4", ".mkv")
         ]
         existing_locs = [loc for loc in potential_locs if os.path.exists(loc)]
-        timestamps_path_lst = glob.glob(os.path.join(
-            self.g_pool.rec_dir, "eye{}*_timestamps.npy".format(eye_id)))
-        timestamp_len = sum(len(np.load(t)) for t in timestamps_path_lst)
-
         if not existing_locs:
             logger.error("no eye video for eye '{}' found.".format(eye_id))
             self.detection_status[eye_id] = "No eye video found."
             return
-        if not timestamps_path_lst:
+        rec, file_ = os.path.split(existing_locs[0])
+        set_name = os.path.splitext(file_)[0]
```

Comment:
What do you think about a utility function to extract the set name for a given path?